js/cookies.js

```diff
@@ -0,0 +1,40 @@
+/* ========================================================================
+ * Bootstrap: cookies.js v3.3.5
+ * ========================================================================
+ * Copyright 2011-2015 Twitter, Inc.
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+ * ======================================================================== */
+
+
++function ($) {
+  'use strict';
+
+    // MANAGING COOKIES
+  	// ================
+
+  	function createCookie(name,value,days) {
+		if (days) {
+			var date = new Date();
+			date.setTime(date.getTime()+(days*24*60*60*1000));
+			var expires = "; expires="+date.toGMTString();
+		}
+		else var expires = "";
```

Comment:
Expected '{' and instead saw 'var'.<br>'expires' is already defined.<br>Mixed double and single quotes.<br>'expires' is defined but never used.